audio/libid3tag/Portfile

```diff
@@ -1,22 +1,24 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem 1.0
+PortSystem	1.0
+PortGroup       github 1.0
 
 name		libid3tag
-version		0.15.1b
+version		0.15.1c
```

Comment:
One more comment, apologies for not catching this earlier: `github.setup` automatically sets `name` and `version`.

No reason to have duplicate fields.

You can remove `name` and `version` and move your `github.setup` up here.

Reply:
done